osu.Game.Rulesets.Osu/Objects/Drawables/DrawableRepeatPoint.cs

```diff
@@ -15,6 +16,7 @@ public class DrawableRepeatPoint : DrawableOsuHitObject, ITrackSnaking
     {
         private readonly RepeatPoint repeatPoint;
         private readonly DrawableSlider drawableSlider;
+        private bool isEndRepeat => repeatPoint.RepeatIndex % 2 == 0;

```

Comment:
Any repeat could be the last one though. Or this is confusing me. 😕

Reply:
This represents in what side the release point is, false for starting side (first control point), true ending side (last control point), was used before by itself
Might be bad wording tho

Reply:
maybe `isRepeatAtEnd`, along with an xmldoc?